js/menus/menuTemplate.js

```diff
@@ -542,19 +509,26 @@ function getTitleBarStyle() {
  * to relaunch application
  *
  * @param app
+ * @param isNativeStyle
  */
-function titleBarActions(app) {
+function titleBarActions(app, isNativeStyle) {
     const options = {
         type: 'question',
         title: i18n.getMessageFor('Relaunch Application'),
-        message: i18n.getMessageFor('Updating Title bar style requires Symphony to relaunch'),
+        message: i18n.getMessageFor('Updating Title bar style requires Symphony to relaunch.'),
+        detail: i18n.getMessageFor('Note: When Hamburger menu is disabled, you can trigger the main menu by pressing the Alt key.'),
         buttons: [i18n.getMessageFor('Relaunch'), i18n.getMessageFor('Cancel')],
         cancelId: 1
     };
     electron.dialog.showMessageBox(electron.BrowserWindow.getFocusedWindow(), options, function (index) {
         if (index === 0) {
-            app.relaunch();
-            app.exit();
+            updateConfigField('isCustomTitleBar', !!isNativeStyle)
+                .then(() => {
+                    app.relaunch();
+                    app.exit();
+                }).catch((e) => {
+                    log.send(logLevels.ERROR, `Unable to update custom title bar settings error: ${e}`);
```

Comment:
can you update this message to say `Unable to disable / enable hamburger menu due to error`